src/main/java/christmas/domain/eventplanner/EventBadge.java

```diff
@@ -0,0 +1,34 @@
+package christmas.domain.eventplanner;
+
+import java.util.Arrays;
+import java.util.Comparator;
+
+public enum EventBadge {
+    SANTA(20_000, "산타"),
+    TREE(10_000, "트리"),
+    STAR(5_000, "별"),
+    NONE(0, "없음");
+
+    private final int threshold;
+    private final String name;
+
+    EventBadge(int threshold, String name) {
+        this.threshold = threshold;
+        this.name = name;
+    }
+
+    public int getThreshold() {
+        return threshold;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public static EventBadge fromBenefitAmount(int benefitAmount) {
+        return Arrays.stream(EventBadge.values())
```

Comment:
```suggestion
        return EventBadge.values().stream()
```
values는 Array가 아닌 Collections를 반환하기 때문에, Arrays.stream()을 사용하는 것 보단 디폴트 메서드인 stream을 사용하는게 더 적합해보여요.